src/Angor/Shared/Services/RelayService.cs

```diff
@@ -135,13 +135,18 @@ public Task LookupDirectMessagesForPubKeyAsync(string nostrPubKey, DateTime? sin
                 _subscriptionsHandling.TryAddRelaySubscription(subscriptionKey, subscription);
             }
 
-            nostrClient.Send(new NostrRequest(subscriptionKey, new NostrFilter
+            var nostrFilter = new NostrFilter
             {
                 P = new[] { nostrPubKey },
                 Kinds = new[] { NostrKind.EncryptedDm },
                 Since = since,
                 Limit = limit
-            }));
+            };
+
+            if (senderNpub != null)
```

Comment:
Why?

Reply:
So we can filter by the sender of the DM if we want to or just all DM to a pubkey